pulsar-broker/src/test/java/org/apache/pulsar/broker/service/PersistentTopicTest.java

```diff
@@ -2251,7 +2251,7 @@ private ByteBuf getMessageWithMetadata(byte[] data) {
     @Test
     public void testGetReplicationClusters() throws MetadataStoreException {
         PersistentTopic topic = new PersistentTopic(successTopicName, ledgerMock, brokerService);
-        topic.initialize();
+        topic.initialize().join();
```

Comment:
do we have a global timeout for single test?

Reply:
Yes we do. https://github.com/apache/pulsar/blob/33f40f6154ad20ffadd954d2fbb9fd1ee4268efb/buildtools/src/main/java/org/apache/pulsar/tests/AnnotationListener.java#L49-L52